src/components/preview/previewIcons.ts

```diff
@@ -5,6 +5,7 @@
  */
 import * as vars from '../../styles/variables';
 import IconFileAudio from '../../icon/content/FileAudio32';
+import IconFileCanvas from '../../icon/content/FileCanvas32';
 import IconFileBoxNote from '../../icon/content/FileBoxNote32';
```

Comment:
minor but we should sort these imports.

```suggestion
import IconFileBoxNote from '../../icon/content/FileBoxNote32';
import IconFileCanvas from '../../icon/content/FileCanvas32';
```